README.md

```diff
@@ -6,7 +6,37 @@ Common Github Actions workflows
 Workflows v2 and above are using Github/AWS OpenID connect to perform the AWS authentication ([refs](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services)).  
 If your repository needs to access AWS you will need to create a dedicated CI profile in [infra-ci](https://github.com/sencrop/infra-ci).
 
-## Standard workflows
+# Standard workflows
+### General purpose
+
+### Version
+
+This workflow will output a version for the current build. It can be either based on the current git commit sha or the current git tag.
+You can the get use the computed version in subsequent jobs using `${{ needs.version.outputs.version }}`.
+
+```yaml
+jobs:
+  version:
+    uses: sencrop/github-workflows/.github/workflows/version-v3.yml@master
+    with:
+      from: sha | tag
```

Comment:
We should maybe make explicit which one we should choose between `sha` and `tag` by default for example in the case of a new repo and not the migration of an existing one where we'd pick whatever existed before ?